scrapper_config.json

```diff
@@ -1,4 +1,14 @@
 {
-    "seed_urls": [],
-    "total_articles_to_find_and_parse": 0
+    "seed_urls": [
+        "https://vestnik.kemsu.ru/jour/issue/view/105/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/104/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/103/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/102/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/101/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/100/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/99/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/98/showToc",
+        "https://vestnik.kemsu.ru/jour/issue/view/97/showToc"
+    ],
+    "total_articles_to_find_and_parse": 10
```

Comment:
ensure that you will be able to demonstrate that you are able to download 100 articles

Reply:
Ensured